### PR TITLE
Mobile trackers release 1.1.0

### PR DESCRIPTION
**Pull request by mhadam**

### CODE REVIEW

File: schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an event where a mobile application is installed.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "application_install",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
```
Comment (chuwy) [inline schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0]: Technically, it is a valid event, but are we sure there's absolutlely nothing to add?

File: schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an event where a mobile application is installed.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
```
Comment (chuwy) [inline schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0]: I'm not sure if I like or not this overly generic semantics. It is not related to `com.snowplowanalytics.`, but at the same time there's no other vendor that can match such a generic event.

File: schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            "description": "The name of the screen viewed."
+        },
+        "type": {
+            "type": "string",
+            "description": "The type of screen that was viewed e.g feed / carousel."
+        },
+        "id": {
+            "type": "string",
+            "maxLength": 36,
+            "description": "An ID from the associated screenview event."
```
Comment (chuwy) [inline schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0]: Is it UUID?

Comment (mhadam) [reply schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0]: Yes. Do you think it's worth making it UUID instead?

File: schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0 (hunk 1 above)
Comment (mhadam) [reply schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0]: Yes, pretty sure this is all we need.

File: schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0 (hunk 2 above)
Comment (mhadam) [reply schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0]: @yalisassoon any thoughts?

File: schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0 (hunk 3 above)
Comment (chuwy) [reply schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0]: Yep, there's `format: uuid`

File: schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0 (hunk 2 above)
Comment (yalisassoon) [reply schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0]: Let's keep it `com.snowplowanalytics.mobile`

**Comment by mhadam** [comment]
@chuwy only question - do I have to add null to any field that isn't required?

File: schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0 (hunk 3 above)
Comment (mhadam) [reply schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0]: just updated all fields where uuid applies

**Comment by chuwy** [comment]
Closing in favor of https://github.com/snowplow/iglu-central/pull/938

**Pull request closed without merging**